packages/apps/docs/src/docs/deploy/chainweb/compact-a-database.md

```diff
@@ -1,103 +1,121 @@
+   compact --from ~/.local/share/chainweb-node/testnet04 --to ~/.local/share/chainweb-node/compact-db --log-dir /tmp/compaction-log-files
    ```
 
-   Creating a backup copy of the database requires you to have more disk space available to store both the active database and the backup until you complete the compaction.
-   However, creating a backup ensures that the node continues to run uninterrupted while the database is being compacted.
-   The backup also ensures tht you can restore the database if something goes wrong with the compacted database.
-
-4. Compact your backup `sqlite` database by running the `cwtool compact` command with the following arguments:
+   Note that the location of the Chainweb root database directory—`~/.local/share/testnet04` in this example—depends on the configuration of the node.
+   If you haven't specified a location in the configuration file, the default location is `~/.local/share/chainweb-node/{chainweb-network-id}`, for example `~/.local/share/chainweb-node/testnet04` for a node in the Kadena test network.
 
-   - `--target-blockheight` to keep enough state available to continue validating blocks after the database has been compacted. An appropriate value depends on the network your node is connected to. For example, if you are compacting your local development network for testing purposes, you would set a lower block height than a node connected to the Kadena test network (`testnet`) or main network (`mainnet`). For a node connected to `mainnet`, it's recommended to use a value of about 4.6 million ("4600000"). In the near future, this option will be removed, and this value will be computed automatically.
-   - `--pact-database-dir` to specify the path to the backup pact `sqlite` state where all of the `*.sqlite`  files are located.
-   - `--log-dir` to specify the directory where you want `cwtool compact` to put the log files it creates, one for each chain. If the directory doesn’t exist, `cwtool compact` creates it. These logs can be useful if something goes wrong.
+   If your node isn't synchronized with the current block height of the network or doesn't have enough history to ensure proper validation, you might see the `compact` operation fail with any error similar to the following:
 
-   For example, run a command similar to the following:
    ```bash
-   cwtool compact \
-     --target-blockheight 4600000 \
-     --pact-database-dir /tmp/sqlite-backup-dd-mm-yyyy \
-     --log-dir /tmp/compact-sqlite-logs
+   2024-08-09T20:03:38.215Z [Error] [] locateLatestSafeTarget: Not enough history to safely compact. Aborting.
    ```
 
-5. Stop your node, replace the original pact `sqlite` state with the newly compacted `sqlite` state from the backup by running a command similar to the following:
+   If you have enough history for compaction to succeed, you should see a message similar to the following:
+
+   ```text
+   2024-08-10T04:08:55.955Z [Debug] [] Latest Common BlockHeight: 9419
```

Comment:
I would use output from a non-buggy invocation of the compact tool. i think that you might have been missing the `--chainweb-version` flag in your invocation, so some of the blockheights are all messed up (see how the earliest blockheight is so much higher)